### DISCUSSION TITLE
Keyer for disk cache

### DISCUSSION THREAD

**Discussion by Mickle-Liutov** [coil-kt/coil#1229] [Q&A]
Hello there, 

As far as I understand, the Keyer is only used for creating the memory cache key, so the question is if it's possible to use it for the disk cache key in a similar way, or it can only be explicitly specified in the load request. If not, I think it would be nice to make it possible.

**Comment by eygraber** [comment]
@colinrtwhite just ran into this as well. Is there a specific reason it can't be done, or was it just not implemented?

**Reply by colinrtwhite** [reply]
@eygraber It hasn't been implemented. Generally I'm a bit more apprehensive to add hooks into the disk cache as any changes are persisted for the long term vs. the memory cache which is cleared on app close. Is [this](https://github.com/coil-kt/coil/issues/1920) what you'd be looking for?
